question_4.py

```diff
@@ -0,0 +1,62 @@
+print("1. Reset all spaces in the car park")
```

Comment:
Can this file be removed from the PR.  I would like this to just contain the README